tox.ini

```diff
@@ -1,13 +1,20 @@
 [tox]
-envlist = py{27,35,36}-celery{3,4},py37-celery4
+envlist =
+    py{27,35,36}-celery{3,4}
+    py37-celery4
+    py{27,35,36,37}-missing_modules
 skip_missing_interpreters = true
 
 [testenv]
+setenv =
+    missing_modules: TEST_MISSING_MODULES_HANDLING=1
 deps =
     celery3: celery~=3.0
     celery4: celery~=4.0
-    -rrequirements-test.txt
```

Comment:
Where DOES it get installed now?

Reply:
Here it is ([lines 15-16](#diff-b91f3d5bd63fcd17221b267e851608e8R15-R16))

```ini
extras =
    !missing_modules: tests
```

Previously, it was
```ini
extras =
    tests
```

So `-rrequirements-test.txt` was redundant.